MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/MultipleGeometriesActivity.java

```diff
@@ -0,0 +1,135 @@
+package com.mapbox.mapboxandroiddemo.examples.annotations;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.style.layers.CircleLayer;
+import com.mapbox.mapboxsdk.style.layers.FillLayer;
+import com.mapbox.mapboxsdk.style.layers.Filter;
+import com.mapbox.mapboxsdk.style.layers.PropertyFactory;
+import com.mapbox.mapboxsdk.style.sources.GeoJsonSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class MultipleGeometriesActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  private MapView mapView;
+  private MapboxMap mapboxMap;
+  private static final String GEOJSON_SOURCE_ID = "GEOJSON FILE";
```

Comment:
Though maybe not an API requirement, I prefer not including spaces in ID names.

Reply:
fixed